product_docs/docs/tpa/23/rel_notes/tpa_23.24_rel_notes.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Trusted Postgres Architect 23.24 release notes 
+navTitle: "Version 23.24"
+---
+
+Released: 17 Oct 2023
+
+Please note that 2ndQuadrant/ansible is now deprecated and `tpaexec setup` now defaults to Community Ansible. 
+Support for using the 2ndQuadrant Ansible fork in will be removed from TPA in April 2024 and the GitHub repository will be archived. 
+You should switch to Community Ansible, which is now the default. For the vast majority of users, this change will be transparent. 
+However, if you are using `--skip-tags` with 2ndQuadrant Ansible, be aware that this is not supported with TPA and Community Ansible. 
+We plan to provide an alternative to `--skip-tags` compatible with Community Ansible before the removal of 2ndQuadrant Ansible.
+
+New features, enhancements, bug fixes, and other changes in Trusted Postgres Architect 23.24 include the following:
+
+| Type | Description |
+| ---- |------------ |
+| Change | `tpaexec setup` now defaults to using community ansible rather than 2ndQuadrant ansible. The option `--use-2q-ansible` can be used to force the use of 2ndQuadrant ansible, which is now deprecated and will be removed in a future release. If you are using `--skip-tags`, see [the install documentation](INSTALL.mdx). |
+| Change | When a repository has been removed from `edb_repositories` in config.yml, `tpaexec deploy` now removes it from the nodes. |
+| Change | TPA will now detect when harp-proxy and harp-manager are running on the same node and use a different config file for harp-proxy. |
+| Change | When using local repositories, TPA will now update the instance local repositories from the one on the TPA host as part of the `upgrade` command. |
```

Comment:
Something doesn't feel right about this particular sentence

Reply:
Which one, you've got four lines...

Reply:
```
When using local repositories, TPA will now update the instance local repositories from the one on the TPA host as part of the `upgrade` command.
```
☝️

Reply:
maybe say "... TPA will now update local repositories on target instances ..."?

Reply:
Its a very hard read...

As part of the `upgrade` command, TPA will update instances using local repositories to use the one on the TPA host*?

* host? The one specified in the config? A repo hosted on the host?